content/en/blog/_posts/get-started-with-kubernetes-using-python.md

```diff
@@ -0,0 +1,217 @@
+
+### Create a Dockerfile
+Now that you have verified the source code works, the first step in containerizing the application is to create a Dockerfile.
+
+In the hello-python/app directory, create a file named Dockerfile with the following contents and save it:
+```
+FROM python:3.7
+
+RUN mkdir /app
+WORKDIR /app
+ADD . /app/
+RUN pip install -r requirements.txt
+
+EXPOSE 5000
+CMD ["python", "/app/main.py"]
+```
+
+This file is a set of instructions Docker will use to build the image. For this simple application, Docker is going to:
+
+1. Get the official [Python Base Image](https://hub.docker.com/_/python/) for version 3.7 from Docker Hub.
+2. In the image, create a directory named app.
+3. Set the working directory to that new app directory.
+4. Copy the local directory’s contents to that new folder into the image.
+5. Run the pip installer (just like we did earlier) to pull the requirements into the image.
+6. Configure port 5000 to be open.
+7. Configure the starting command to use when the container starts.
+
+### Create an image
+At your command line or shell, in the hello-python/app directory, build the image with the following command:
+```
+docker build -f Dockerfile -t hello-python:latest .
+```
+
+This will perform those seven steps listed above and create the image. To verify the image was created, run the following command:
+
+```
+docker image ls
+```
+<img src="/images/blog/get-started-with-kubernetes-using-python/docker-image-ls.png" alt="Docker image listing" />
+
+The application is now containerized, which means it can now run in Docker and Kubernetes!
+
+## Running in Docker
+Before jumping into Kubernetes, let’s verify it works in Docker.
+Run the following command to have Docker run the application in a container and map it to port 5001:
+```
+docker run -p 5001:5000 hello-python
+```
+Now navigate to http://localhost:5001, and you should see the “Hello form Python!” message. 
+
+### More info
+* [Get started with Docker](https://docs.docker.com/get-started/)
+* [Best practices for writing Dockerfiles](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/)
+* [Docker Cheat Sheet](https://www.docker.com/sites/default/files/Docker_CheatSheet_08.09.2016_0.pdf) (pdf)
+
+## Running in Kubernetes
+You are finally ready to get the application running in Kubernetes. Because you have a web application, you will create a service and a deployment.
+
+First verify your kubectl is configured. At the command line, type the following:
+```
+kubectl version
+```
+
+If you don’t see a reply with a Client and Server version, you’ll need to [install](https://kubernetes.io/docs/tasks/tools/install-kubectl/) and configure it.
+
+Make sure it is using the Docker for Desktop context by running the following:
+```
+kubectl config use-context docker-for-desktop
```

Comment:
I'm not sure this is needed or recommended for readers not using Windows (or Mac?). Perhaps worth prefacing the above with "If you are running on Windows or Mac, make sure.."

Reply:
Ok, I'm definitely coming from a windows point of view, so thanks that helps broaden the message.